Mathlib/Algebra/Order/Field/Pi.lean

```diff
@@ -1,5 +1,5 @@
 /-
-Copyright (c) 2014 Robert Lewis. All rights reserved.
+Copyright (c) 2014 Robert Y. Lewis. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies
```

Comment:
That's funny. Why am I the author but not the copyright holder?

Reply:
I'm not sure: maybe a weird previous split?

Reply:
🤷

Reply:
See leanprover-community/mathlib#17350

Reply:
BTW, @YaelDillies am I right that the only lemma in this file doesn't need a (semi)field structure, only a linear ordered additive commutative group with dense order? If yes, could you please generalize it and move the file elsewhere?

Reply:
You are absolutely right: #15327. Not sure where to move the lemma though